docs/setup/upgrade.asciidoc

```diff
@@ -1,45 +1,55 @@
 [[upgrade]]
-== Upgrading Kibana
+== Upgrading {kib}
+
 
 [IMPORTANT]
 ===========================================
-Before upgrading Kibana:
+Before upgrading {kib}:
 
 * Consult the <<breaking-changes,breaking changes>> docs.
 * Test upgrades in a dev environment before upgrading your production servers.
-* Backup your data using the Elasticsearch
-  {ref}/modules-snapshots.html[snapshots] feature.
+* Backup your data using the {es} {ref}/modules-snapshots.html[snapshots] feature.
   You **cannot roll back** to an earlier version unless you have a backup of
   your data.
 * If you are using custom plugins, check that a compatible version is
   available.
 ===========================================
 
-Depending on which version of Kibana you're upgrading from, the upgrade process
-will vary. Please consult the following table to determine which process you
-should follow:
-
-[cols="1<m,1<m,3",options="header",]
-|=======================================================================
-|Upgrade From   |Upgrade To     |Supported Upgrade Type
-|3.x            |5.x            |<<upgrade-new-install,New installation>>
-|4.0 or 4.1     |5.x            |<<upgrade-standard-reindex,Standard upgrade & reindex>>
-|4.x >= 4.2     |5.x            |<<upgrade-standard,Standard upgrade>>
-|5.0.0 pre GA   |5.x            |<<upgrade-standard-reindex,Standard upgrade & reindex>>
-|5.x            |5.y            |<<upgrade-standard,Standard upgrade>> (where `y > x`)
-|5.x            |6.x            |<<upgrade-standard,Standard upgrade>>
-// & <<migrating-6.0-index,Reindex transformation>>
-|6.x            |6.y            |<<upgrade-standard,Standard upgrade>> (where `y > x`)
-|6.x            |7.x            |<<upgrade-standard,Standard upgrade>>
-|=======================================================================
-
-NOTE: Saved searches, visualizations, and dashboards created in Kibana 4.x or 5.x
-can be generally imported into 6.x.
+Depending on which version of {kib} you're upgrading from, the upgrade process to 7.0
+will vary. 
 
-include::upgrade/upgrade-standard.asciidoc[]
+The recommended path is to upgrade to 6.7 before upgrading to 7.0. 
+This makes it easier to identify the changes you need to make to upgrade and enables 
+you to perform a rolling upgrade with no downtime. 
+
+Please see 
+*{stack-ref}/upgrading-elastic-stack.html[Upgrading the Elastic Stack]* for a 
+comprehensive overview of the upgrade process.
+
+[float]
+==== Upgrading from 6.6 or earlier
+{es} can read indices created in the previous major version. If you have indices 
+created in 5.x or before, you must reindex or delete them before upgrading to 7.0.0.
+See {stack-ref}/upgrading-elastic-stack.html#oss-stack-upgrade[Upgrading from 6.6 or earlier] 
+for more information.
+
+Once your reindex is complete, you can follow the <<upgrade-standard, Standard upgrade>> 
+instructions. The recommended path is to upgrade to 6.7 before upgrading to 7.0.
```

Comment:
I think it would be good to call out here that this manual reindex process is not necessary if they upgrade to 6.7 and use the Upgrade Assistant instead.

Reply:
@joshdover Wouldn't someone on a much earlier version (say, 4.x) need to go through a reindex to be able to get to 6.7, before they could use the Upgrade Assistant to prepare for the upgrade to 7.0?

Reply:
They will, but they also will have to through 5.6 first as well. If you're on 2.x for instance, the process would be
- Reindex 1.x index
- Upgrade to 4.x
- Reindex every 2.x index
- Upgrade to 5.6
- Reindex every 4.x index
- Upgrade to 6.7
- Reindex every 5.x index

I think the linked stack doc makes it pretty clear that upgrading directly from 4.x isn't supported:
> Elasticsearch supports rolling upgrades between minor versions, from Elasticsearch 5.6 to 6.7, and from 6.7 to 7.0.0-rc1.

In that case, the customer should be first upgrading and following the docs for 4.x to 5.6. Once they're on 5.6, they can use the upgrade assistant in 5.6 to reindex without having to do this manually. I'm not sure the best way to convey this in the documentation besides linking to the docs for upgrading to 5.6 first?

Reply:
Right, okay - so maybe noting with a link that an Upgrade Assistant is available in 5.6? As you note, I want to avoid detailing every upgrade scenario, and assume that people will reference the upgrade docs from previous versions (if that is their starting point).

Reply:
Thanks - pushed some updates